docs/ref/models/database-functions.txt

```diff
@@ -783,6 +783,32 @@ It can also be registered as a transform. For example::
     >>> # Get vectors whose arccosine is less than 1
     >>> vectors = Vector.objects.filter(x__acos__lt=1, y__acos__lt=1)
 
+``ACosh``
+---------
+
+.. class:: ACosh(expression, **extra)
+
+.. versionadded:: 3.2
+
+Returns the inverse hyperbolic cosine of a numeric field or expression. The
+expression value must be greater than or equal to 1.
+
+Usage example::
+
+    >>> from django.db.models.functions import ACosh
+    >>> Vector.objects.create(x=1.5, y=3.9)
+    >>> vector = Vector.objects.annotate(x_acosh=ACosh('x'), y_acosh=ACosh('y')).get()
+    >>> vector.x_acosh, vector.y_acosh
+    (0.9624236501192069, 2.0372664661418107)
+
+It can also be registered as a transform. For example::
+
+    >>> from django.db.models import FloatField
+    >>> from django.db.models.functions import ACosh
+    >>> FloatField.register_lookup(ACosh)
+    >>> # Get vectors whose inverse hyperbolic cosine is less than 1
+    >>> vectors = Vector.objects.filter(x__acosh__lt=1, y__acosh__lt=1)
```

Comment:
There are so many similar examples. I think it's better to extracting them to a section "Registering a function as a transform" which walks through just one of them. Then each section can say just "It can also be registered as a transform." with a link.

Reply:
I think that we could follow up in a separate PR for that later. I agree it is rather repetitive. (Also tedious to write!)

Reply:
Okay please remember to come back to this or make a ticket :)